andela/nazgul-rc
# #156958156: Shop ratings and reviews

#### What does this PR do?
Integrates shop ratings and reviews into app

#### Description of Task to be completed?

- Allow a user who has made a purchase to rate and review the shop
- Allow user to overwrite his/her rating and review of the shop
- Allow user and admin to see average rating of the shop
- Allow user and admin to read reviews of the shop

#### How should this be manually tested?

- npm run git clone `https://github.com/andela/nazgul-rc.git` to clone the repository
- navigate to the nazgul-rc folder
- run `curl https://install.meteor.com/ | sh`
- run `npm install -g reaction-cli`
- install dependencies with`meteor npm install`
- run `reaction` to start up server
- once app is running, navigate to the app using localhost:3000
- navigate to shop page
- click on `Shop Reviews`
- see average rating for the shop
- see reviews for the shop
- log in and make a purchase
- click on `Rate Us`
- rate and review the shop

#### What are the relevant pivotal tracker stories?
#156958156
